csrc/transformer/normalize_kernels.cu

```diff
@@ -1069,8 +1069,8 @@ __global__ void LayerNormBackward2(const float* out_grad,
 
     int row = blockIdx.x;
     int id = threadIdx.x;
-    int wid = id / WARP_SIZE;
-    int warp_num = (THREADS < row_stride ? THREADS : row_stride) / WARP_SIZE;
+    int wid = id >> 5;

```

Comment:
Please replace the `5` with a descriptively named variable. Thanks.

Reply:
got it. thanks